setup.cfg

```diff
@@ -104,3 +104,7 @@ ignore_errors = True
 # todo: add proper typing to this module...
 [mypy-torchmetrics.retrieval.*]
 ignore_errors = True
+
+# todo: add proper typing to this module...
+[mypy-torchmetrics.audio.*]
```

Comment:
Instead of ignoring it right from the start, can't we make sure it is typed? Otherwise we will probably never do it.

Reply:
@quancs please ^^

Reply:
OK. I will try.